packages/repeater/src/api/HttpRepeatersManager.spec.ts

```diff
@@ -0,0 +1,77 @@
+import 'reflect-metadata';
+import {
+  CreateRepeaterRequest,
+  DeleteRepeaterRequest,
+  ListRepeatersRequest
+} from './commands';
+import { HttpRepeatersManager } from './HttpRepeatersManager';
+import { RepeatersManager } from './RepeatersManager';
+import { CommandDispatcher } from '@secbox/core';
+import { anyOfClass, instance, mock, reset, verify, when } from 'ts-mockito';
+
+describe('HttpRepeatersManager', () => {
+  const MockedCommandDispatcher = mock<CommandDispatcher>();
+  let manager!: RepeatersManager;
+
+  beforeEach(() => {
+    manager = new HttpRepeatersManager(instance(MockedCommandDispatcher));
+  });
+
+  afterEach(() => reset(MockedCommandDispatcher));
+
+  describe('createRepeater', () => {
+    it('should create repeater', async () => {
+      when(
+        MockedCommandDispatcher.execute(anyOfClass(CreateRepeaterRequest))
+      ).thenResolve();
+
+      when(
+        MockedCommandDispatcher.execute(anyOfClass(ListRepeatersRequest))
+      ).thenResolve([
+        { name: 'bar', id: '142' },
+        {
+          name: 'foo',
+          id: '42'
+        }
+      ]);
+
+      const result = await manager.createRepeater({ name: 'foo' });
+
+      verify(
+        MockedCommandDispatcher.execute(anyOfClass(CreateRepeaterRequest))
+      ).once();
+      verify(
+        MockedCommandDispatcher.execute(anyOfClass(ListRepeatersRequest))
+      ).once();
+      expect(result.repeaterId).toBe('42');
```

Comment:
It is better to verify the entire result: 

```suggestion
      expect(result).toMatchObject({ repeaterId: '42' });
```